File: data/mi1/backup/items/lookout.lua
-- the local objects
items["lookout.stairs"] = {
	text = strings.objects.stairs,
	pos = {230, 0, 0},
	walk_to = {250, 0}, 
	face = "south",
	size = {30, 10},			
	actions = {
		walk = ms { {action.change_room, {id=1,room="village1"}}}
	}
}

items["lookout.path"] = {
	text = strings.objects.path,
	pos = {294, 0, 0},
	walk_to = {320, 50}, 
	face = "west",
	size = {26, 144},			
	actions = {
		walk = ms { {action.change_room, {id=1,room="meleemap"}}}
	}
}


items["lookout.lookout"] = {
	tag = "lookout",
	text_offset = {0, 60},
    text_color = {170, 170, 170, 255},
	text = strings.objects.lookout,
	pos = {114, 36, 0},
	walk_to = {134, 36},
	size = {20, 47},
	offset = {-10,0},
	face = "west",
	model ="lookout",
	anim ="idle_right",
	flip = true,
	actions = {
		talk = curry(start_dialogue, {dialogue = "lookout"})
	}
}

items["lookout.fire"] = {
	pos = {126, 52, -1},
	model = "fire",
    anim = "default"
}

File: data/mi1_py/rooms/lookout_.py
from lib_py.scumm.scumm import State
from lib_py.scumm.entity import Item, CharItem
import lib_py.scumm.scripts as ssc
import lib_py.engine as engine
import mi1_py.variables as var

st = engine.data['strings']
sl = st['dialogues']['lookout']

State.addItem (Item(
    id = 'lookout.fire',
    model = 'fire', 
    pos = (126, 52)))

State.addItem (Item(
    id = 'lookout.stairs', 
    text = st['objects']['stairs'],
    pos = (230, 0),
    width = 30, 
    height = 10, 
    walkto = var.lookout_stairs, 
    dir = 's',
    actions = {
        'walkto': ssc.changeRoom(room = 'village1', pos = [8, 71, 0], dir='s')
    }))

State.addItem (CharItem (
    id = 'lookout',
    text = st['objects']['lookout'],
    model='lookout', 
    pos = (114, 36),
    width = 20, 
    height = 47, 
    offset= [-10,0],
    walkto = [134, 36], 
    dir = 'w', 
    state='idle', 
    text_color=[170, 170, 170, 255], 
    text_offset=[0,60],
    chardir = 'w',
    speed = 100,
    actions = {
        'look': ssc.say(lines=[st['dialogues']['lookout'][50]]),
        'talkto': ssc.startDialogue('lookout')
    }
))

# State.addItem (Item(
#     id = 'lookout.lookout', 
#     text = st['objects']['lookout'],
#     width = 20, 
#     height = 47, 
#     offset= [-10,0],
#     walkto = [134, 36], 
#     dir = 'w',


